docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -775,9 +775,89 @@ See below for a visual representation:
 
 **Birth Weight** 
 
-.. todo:: 
+Our model will apply the effect size of maternal consumption of iron fortified 
+foods on infant birth weight under the following assumptions:
+
+1. The effect of consuming iron fortified foods takes place six months after 
+the onset of consumption. This assumption is partially supported by Andersson 
+et al. 2010, which found that the peak response occurs at approximately six 
+months after onset; however, we simplified this finding for our model and 
+assumed that there was only a response six months after onset of consumpiton 
+of iron fortified foods.
+
+2. The effect of maternal iron fortified food consumption must occur at the 
+start of the third trimester of pregnancy in order to impact infant birth 
+weight, which we assumed was equal to 3 months prior to birth of the infant. 
+We chose to make this assumption because the effect of maternal iron was 
+measured at the third trimester in the Haider et al. 2013 paper.
+
```

Comment:
I think this can start earlier (7 weeks) when plasma expansion occurs and there is need for iron stores.

Verhoef, H., Mwangi, M.N., Cerami, C. et al. Antenatal iron supplementation and birth weight in conditions of high exposure to infectious diseases. BMC Med 17, 146 (2019). https://doi.org/10.1186/s12916-019-1375-9

Reply:
See my comments in the thread above